### PR TITLE
Update the Android SDK manual install to reflect the removal of ACRA.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.clubhouse.io/buddybuild/story/8570/update-android-manual-sdk-integration-steps-we-removed-acra

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://132-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://132-bb-docs.surge.sh

File: sdk/integration.adoc [hunk 1]
```diff
@@ -190,14 +190,13 @@ repositories {
...
 
     maven {
-        url 'https://dl.bintray.com/buddybuild/maven'
+      url 'https://dl.bintray.com/buddybuild/maven'
     }
 }
 
 dependencies {
-    compile 'com.buddybuild:sdk:0.0.84@aar'
+    compile 'com.buddybuild:sdk:0.1.9@aar'
```
Comment (rasharab) [inline sdk/integration.adoc]: This is now 0.1.12

File: sdk/integration.adoc [hunk 2]
```diff
@@ -208,7 +207,7 @@ Two things to note here:
 . You can put the `repositories` section in the base `build.gradle` file,
   but you need to put it under the `allprojects` section.
 
-. Note the `@aar` at the end of `com.buddybuild:sdk:0.0.84`
+. Note the `@aar` at the end of `com.buddybuild:sdk:0.1.9@aar`
```
Comment (rasharab) [inline sdk/integration.adoc]: 0.1.12.
Sorry, should've told you

**Comment by bbdocs** [comment]
See the website here: http://132-bb-docs.surge.sh

**Pull request merged**